### PR TITLE
Friends list

### PR DESCRIPTION
**Pull request by jonathanAgarcia**
Hey Team. Hoping to get a review for my last dummy/static data PR. I am making this request because I added some functionality that helps connect @chaomonica and my component and It will be helpful for her to have. Thanks everyone.

### CODE REVIEW

File: front-end/src/components/CenterConsole/CenterConsole.js [hunk 1]
```diff
@@ -6,21 +6,10 @@ import { Container, Grid } from '@material-ui/core';
 const CenterConsole = (props) => {
   return (
     <Container maxWidth='sm'>
```
Comment (chaomonica) [inline front-end/src/components/CenterConsole/CenterConsole.js:8]: Just checking again, would it still be okay for me to remove the Container around RecipeList?

File: front-end/src/components/CenterConsole/RecipeList.js [hunk 2]
```diff
@@ -4,14 +4,15 @@ import SingleRecipe from './SingleRecipe.js';
...
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
       recipeList: userCookbook.recipes,
       disableShowMoreButton: false,
       disablePreviousButton: true,
       startOfSlice: 0,
       endOfSlice: 6,
+      selectedCatagorie: this.props.selectedCatagorie
```
Comment (chaomonica) [inline front-end/src/components/CenterConsole/RecipeList.js:15]: category? :P

**Review by chaomonica — APPROVED** [review]
Looks awesome! :D

File: front-end/src/components/CenterConsole/CenterConsole.js (hunk 1 above)
Comment (jonathanAgarcia) [reply front-end/src/components/CenterConsole/CenterConsole.js:8]: hey @chaomonica, That would be fine. It does slightly change the mainPage center console by making the cards only slightly larger(not an issue at all). Sorry I did not include it, but no problem for you to rely on it being removed. I'll leave it removed on my end just to make sure there are no issues going forward

**Pull request merged**